app/src/main/java/com/google/droidjump/StartFragment.java

```diff
@@ -50,15 +55,20 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
     }
 
     private void play(View view) {
-        findNavController(view).navigate(R.id.action_start_screen_to_game_screen);
+        Bundle args = new Bundle();
+        args.putInt(GAME_VIEW_LEVEL_STRING, activity.getCurrentLevel());
+        findNavController(view).navigate(R.id.action_start_screen_to_game_screen, args);
     }
 
     private void chooseLevel(View view) {
         findNavController(view).navigate(R.id.action_start_screen_to_levels_screen);
     }
 
     private void startNewGame(View view) {
-        findNavController(view).navigate(R.id.action_start_screen_to_game_screen);
+        Bundle args = new Bundle();
+        args.putInt(GAME_VIEW_LEVEL_STRING, 1);
```

Comment:
Let's encapsulate the logic of changing the level somewhere. Maybe a helper class could work.

Reply:
Makes sense

Reply:
Fixed